src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -889,6 +971,50 @@ private synchronized void regenerateAITasks() {
 		editorReady = true;
 	}
 
+	private void updateBreedingConditions() {
+		if (breedable.isSelected()) {
+			hasAI.setSelected(true);
+			hasAI.setEnabled(false);
+			this.breedTriggerItems.setEnabled(true);
+			this.tameable.setEnabled(true);
+		} else {
+			hasAI.setEnabled(true);
+			this.breedTriggerItems.setEnabled(false);
+			this.tameable.setEnabled(false);
+		}
+	}
+
+	private void updateTradingConditions() {
+		aiBase.setEnabled(!canTrade.isSelected());
+		breedable.setEnabled(!canTrade.isSelected());
+		ridable.setEnabled(!canTrade.isSelected());
+		villagerTradingType.setEnabled(canTrade.isSelected());
+		wanderingTraderTradingType.setEnabled(canTrade.isSelected());
+		professionTrade.setEnabled(canTrade.isSelected());
+		fullUpdateSound.setEnabled(canTrade.isSelected());
+		emptyUpdateSound.setEnabled(canTrade.isSelected());
+		notificationSound.setEnabled(canTrade.isSelected());
+		restockCondition.setEnabled(canTrade.isSelected());
+		rewardXp.setEnabled(canTrade.isSelected() && !villagerTradingType.isSelected());
+		tradingCondition.setEnabled(canTrade.isSelected());
+		guiBoundTo.setEnabled(!canTrade.isSelected());
+		inventorySize.setEnabled(!canTrade.isSelected());
+		inventoryStackSize.setEnabled(!canTrade.isSelected());
+		canControlForward.setEnabled(!canTrade.isSelected());
+		canControlStrafe.setEnabled(!canTrade.isSelected());
+
+		if (canTrade.isSelected()) {
+			aiBase.setSelectedItem(villagerTradingType.isSelected() ? "Villager" : "(none)");
+			guiBoundTo.setSelectedItem("<NONE>");
+			breedable.setSelected(false);
+			updateBreedingConditions();

```

Comment:
This is probably invocated by one line above already, but verify

Reply:
I double checked, looks good to me

Reply:
So setSelected does not already trigger updateBreedingConditions?